tests/rekey.py

```diff
@@ -14,31 +15,17 @@ def test(network, args):
     network.consortium.rekey_ledger(primary)
     return network
 
-
-# Run some write transactions against the logging app
-def record_transactions(primary, txs_count=1):
-    with primary.node_client() as nc:
-        check_commit = infra.checker.Checker(nc)
-
-        with primary.user_client() as c:
-            for i in range(1, txs_count):
-                check_commit(
-                    c.rpc("LOG_record", {"id": i, "msg": f"entry #{i}"}), result=True
-                )
-
-
 def run(args):
     hosts = ["localhost", "localhost"]
 
     with infra.ccf.network(
         hosts, args.binary_dir, args.debug_nodes, args.perf_nodes, pdb=args.pdb,
     ) as network:
         network.start_and_join(args)
-        primary, _ = network.find_primary()
 
-        record_transactions(primary)
-        test(network, args)
-        record_transactions(primary)
+        network = e2e_logging.test(network, args)
```

Comment:
I wonder whether test "blocks" should include other test blocks or whether only the test suite should be allowed to include blocks. I'm leaning towards the latter and I think the `logging_app` should be used here instead?

Reply:
Yeah I think you're right. I thought this one was fine since its in the `__main__`/`run()` of this test, but it should still be using `logging_app` rather than depending on another suite-block.